civihr_employee_portal/views/templates/views-view-table--Documents--block-1.tpl.php

```diff
@@ -65,11 +65,11 @@
                     <?php foreach ($statuses as $statusKey => $statusValue): ?>
                       <?php
                         $selected = '';
-                        if ($statusKey == (int) strip_tags($content)):
+                        if (strtolower($statusValue) == strip_tags($content)):
                           $selected = ' selected="selected"';
                         endif;
                         ?>
-                      <option value="<?php print $statusKey; ?>"<?php print $selected; ?>><?php print $statusValue; ?></option>
+                      <option value="<?php print $statusKey; ?>"<?php print $selected; ?>><?php print ucwords($statusValue);?></option>
```

Comment:
Instead of 3 different `<?php` parts here, how about moving them into one 

```
<?php printf('<option value="%s" %s>%s</option>', $statusKey, $selected, ucwords($statusValue)); ?>
```